RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -31,12 +31,13 @@
 # SEEDING LAYERS
 from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
 from RecoTracker.IterativeTracking.DetachedTripletStep_cff import detachedTripletStepSeedLayers
-mixedTripletStepSeedLayersA = cms.EDProducer('SeedingLayersEDProducer',
-     layerList = cms.vstring('BPix2+FPix1_pos+FPix2_pos', 'BPix2+FPix1_neg+FPix2_neg'),
-#    layerList = cms.vstring('BPix1+BPix2+BPix3', 
+import RecoTracker.TkSeedingLayers.seedingLayersEDProducer_cfi as _seedmod
+mixedTripletStepSeedLayersA = _seedmod.seedingLayersEDProducer.clone(
```

Comment:
there is now a redundant `import RecoTracker.TkSeedingLayers.seedingLayersEDProducer_cfi as _mod` on L164.
Please remove it there.